sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestInheritedAnnotations.java

```diff
@@ -37,7 +38,7 @@
     @Rule
     public H2DatabaseRule dbRule = new H2DatabaseRule().withPlugin(new SqlObjectPlugin());
 
-    private MockClock mockClock = new MockClock();
+    private MockClock mockClock = new MockClock(ZonedDateTime.now());
```

Comment:
How come we removed the default constructor?  Now all use sites have to do this repeatedly.

Reply:
To make it explicit in the tests themselves that you're relying on whatever the current time is, which is an uncertainty, rather than it being hidden as an implementation detail you could overlook when reading a test.

Reply:
Would factory methods `now()` and `at(...)` suit you?